docs/subnets/reference-elastic-subnets-parameters.md

```diff
@@ -0,0 +1,224 @@
+# Elastic Subnets Parameters
+
+Avalanche Permissioned Subnets turn into Elastic Subnet via the `TransformSubnetTx` transaction.
+`TransformSubnetTx` specifies a set of structural parameters for the Elastic Subnet.
+This reference describes these structural parameters and illustrates the constraints they must satisfy.
+
+## Elastic Subnet Parameters
+
+### `Subnet`
+
+`Subnet` has type `ids.ID` and it's the Subnet ID.
+`Subnet` is the ID of the `CreateSubnetTx` transaction that created the Subnet in the first place. 
+The following constraints apply:
+
+* `Subnet` must be different from `PrimaryNetworkID`.
+
+### `AssetID`
+
+`AssetID` has type `ids.ID` and it's the ID of the asset to use when staking on the Subnet.
+The following constraints apply:
+  
+* `AssetID` must not be the `Empty ID`.
+* `AssetID` must not be `AVAX ID`.
+
+`AVAX ID` is the Primary Network asset.
+
+### `InitialSupply`
+
+`InitialSupply` has type `uint64` and it's the initial amount of `AssetID` in the Subnet.
+The following constraints apply:
+
+* `InitialSupply` must be larger than zero.
+
+### `MaximumSupply`
+
+`MaximumSupply` has type `uint64` and it's the maximum amount `AssetID` that Subnet can hold at any time.
+The following constraints apply:
```

Comment:
```suggestion
`MaximumSupply` has type `uint64` and it's the maximum amount of `AssetID` that Subnet can hold at any time.
The following constraints apply:
```

Reply:
Is this comment correct? What does it mean to have a maximum amount of `AssetID` that Subnet can hold at any time. If the token exists on the subnet, then the VM implementation can determine what amount to create more of. How exactly is this maximum applied?

Reply:
Fixed, but I'd ask @StephenButtolph to cross-check